specification/migrate/resource-manager/Microsoft.OffAzure/stable/2020-01-01/migrate.json

```diff
@@ -0,0 +1,4943 @@
+      "post": {
+        "tags": [
+          "Sites"
+        ],
+        "summary": "Method to get site health summary.",
+        "operationId": "Sites_GetSiteHealthSummary",
+        "consumes": [
+          
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The ID of the target subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "siteName",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "description": "The API version to use for this operation.",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/SiteHealthSummaryCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "Get VMware site health summary.": {
+            "$ref": "./examples/VMwareSiteHealthSummary_Post.json"
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.OffAzure/VMwareSites/{siteName}/vcenters/{vcenterName}": {
+      "get": {
+        "tags": [
+          "VCenter"
+        ],
+        "summary": "Method to get a vCenter.",
+        "operationId": "VCenter_GetVCenter",
+        "consumes": [
+          
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "subscriptionId",
+            "in": "path",
+            "description": "The ID of the target subscription.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "description": "The name of the resource group. The name is case insensitive.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "siteName",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "vcenterName",
+            "i
```

Comment:
readOnly was removed from this field. Do you allow users to change this?

Reply:
It can't be set by user for these resources but can be set in other resources which are not in review in this swagger. Adding it as readonly for now. Will change it later when other features will come.